tasks/k8s.py

```diff
@@ -35,8 +35,28 @@ def mysql_secret(ctx, host, password, port=3306, username = 'spaceship-app', db
 
   utils.k8s_apply(secret, dry_run = False)
 
+@task(
+  help={
+    'host': 'Redis host',
+    'port': 'Redis port (default: 6379)',
+    'db': "Number of the Redis DB (default: '0')",
+  }
+)
+def redis_secret(ctx, host, port=3306, db=0):
```

Comment:
Ohhh, _this_ is how we do this... I spent a long time googling around for what ${eval} meant when creating the secrets for oauth. I should add a method to recreate that, here.

Reply:
what `{$eval: 'stuff'}` means: https://taskcluster.github.io/json-e/#Language/-eval-